packages/www/components/Dashboard/Player/index.tsx

```diff
@@ -1,63 +1,50 @@
-import { useEffect, useRef } from "react";
+import { useRef, useEffect } from "react";
 import { Box } from "@livepeer.com/design-system";
-import muxjs from "mux.js/dist/mux.js";
+import videojs from "video.js";
+import "videojs-contrib-quality-levels";
+import "videojs-hls-quality-selector";
+import "video.js/dist/video-js.css";
 
-const Player = ({
-  src,
-  posterUrl = "https://via.placeholder.com/160x90/1e1e21/1e1e21",
-  config = {},
-}) => {
-  const video: any = useRef(null);
-  const videoContainer: any = useRef(null);
-  const controller: any = useRef({});
+export const Player = (props) => {
+  const videoRef = useRef(null);
+  const playerRef = useRef(null);
+  const { options, onReady } = props;
 
   useEffect(() => {
-    window["muxjs"] = muxjs;
-
-    // Use compiled versions of these libraries so they work with ad blockers
-    const shaka = require("shaka-player/dist/shaka-player.ui.js");
-
-    const player = new shaka.Player(video.current);
-    const ui = new shaka.ui.Overlay(
-      player,
-      videoContainer.current,
-      video.current
-    );
-
-    ui.configure(config);
-
-    // Store Shaka's API in order to expose it as a handle.
-    controller.current = {
-      player,
-      ui,
-      videoElement: video.current,
-      config: {},
-    };
+    // make sure Video.js player is only initialized once
+    if (!playerRef.current) {
```

Comment:
What if `options` or `onReady` changes? Do we really need this check if the `useEffect` memo below supposedly only calls this when one of the memos change? Or is this that one that might have some exceptions? not sure